manifest.yaml

```diff
@@ -3,36 +3,36 @@ dotnet:
     '3.1/aspnet':
       data:
         image_repo: mcr.microsoft.com/dotnet/core/aspnet
-        image_version: 3.1.24
+        image_version: 3.1.27
         image_type: buster-slim
         env_version_key: ASPNETCORE_VERSION
     '3.1/runtime':
       data:
         image_repo: mcr.microsoft.com/dotnet/core/runtime
-        image_version: 3.1.24
+        image_version: 3.1.27
         image_type: buster-slim
         env_version_key: DOTNET_VERSION
     '5.0/aspnet':
       data:
         image_repo: mcr.microsoft.com/dotnet/aspnet
-        image_version: 5.0.16
+        image_version: 5.0.9
         image_type: buster-slim
         env_version_key: ASPNETCORE_VERSION
     '5.0/runtime':
       data:
         image_repo: mcr.microsoft.com/dotnet/runtime
-        image_version: 5.0.16
+        image_version: 5.0.9
```

Comment:
this seems to have gone backwards in versions?

Reply:
Oops, I blame the ordering in https://mcr.microsoft.com/v2/dotnet/runtime/tags/list